src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt

```diff
@@ -196,14 +196,19 @@ class DeliverablesController(
   }
 
   @ApiResponseSimpleSuccess
-  @Operation(summary = "Marks a submission from a project as completed.")
+  @Operation(summary = "Marks a submission from a project as complete or incomplete.")
   @PostMapping("/{deliverableId}/submissions/{projectId}/complete")
   fun completeSubmission(
       @PathVariable deliverableId: DeliverableId,
-      @PathVariable projectId: ProjectId
+      @PathVariable projectId: ProjectId,
+      @Parameter(
+          description =
+              "True if the submission should be marked as Completed, or Incomplete otherwise. Defaults to true. ")
+      @RequestParam
+      isComplete: Boolean?,
```

Comment:
Can we make this a request payload instead? Generally in our API, when we have an endpoint that can update some resource attribute to a client-supplied value, the URL identifies which resource to modify, and the payload says what values to write.

Or alternately -- and I think this might be clearer in terms of naming -- make "mark as incomplete" a separate endpoint so that the "complete" endpoint always marks submissions as complete.